cmake/kokkos_arch.cmake

```diff
@@ -107,6 +107,47 @@ KOKKOS_ARCH_OPTION(FIJI            GPU  "AMD Fiji architecture")
 KOKKOS_ARCH_OPTION(VEGA            GPU  "AMD Vega architecture")
 KOKKOS_ARCH_OPTION(GFX901          GPU  "AMD GFX architecture")
 
+
+IF (KOKKOS_ENABLE_CUDA)
+ #Choose default version numbers if not given
+ IF (KOKKOS_ARCH_KEPLER)
+   SET(KOKKOS_ARCH_KEPLER35 ON)
+ ENDIF()
+ 
+ IF (KOKKOS_ARCH_MAXWELL)
+   SET(KOKKOS_ARCH_MAXWELL50 ON)
+ ENDIF()
+ 
+ IF (KOKKOS_ARCH_PASCAL)
+   SET(KOKKOS_ARCH_PASCAL60 ON)
+ ENDIF()
+ 
+ IF (KOKKOS_ARCH_VOLTA)
+   SET(KOKKOS_ARCH_VOLTA70 ON)
+ ENDIF()
+ 
+ #Regardless of version, make sure we define the general architecture name
+ IF (KOKKOS_ARCH_KEPLER30 OR KOKKOS_ARCH_KEPLER32 OR KOKKOS_ARCH_KEPLER35 OR KOKKOS_ARCH_KEPLER37)
+   SET(KOKKOS_ARCH_KEPLER ON)
+ ENDIF()
+ 
+ #Regardless of version, make sure we define the general architecture name
+ IF (KOKKOS_ARCH_MAXWELL50 OR KOKKOS_ARCH_MAXWELL52 OR KOKKOS_ARCH_MAXWELL53)
+   SET(KOKKOS_ARCH_MAXWELL ON)
+ ENDIF()
+
+ #Regardless of version, make sure we define the general architecture name
+ IF (KOKKOS_ARCH_PASCAL60 OR KOKKOS_ARCH_PASCAL61)
+   SET(KOKKOS_ARCH_PASCAL ON)
+ ENDIF()
+
+  #Regardless of version, make sure we define the general architecture name
+  IF (KOKKOS_ARCH_VOLTA70 OR KOKKOS_ARCH_VOLTA72)
+    SET(KOKKOS_ARCH_VOLTA ON)
+  ENDIF()
+ENDIF()
```

Comment:
Can you confirm that we do not want to change the cache for these?

Reply:
See below.

Reply:
We may want to eliminate Kokkos_ARCH_MAXWELL, etc, without a version number so that we don't have the problem of multiple cache variables causing the same behavior. Leaving for now to match Makefile.kokkos